components/aggregation/mat/CustomTooltip.js

```diff
@@ -7,45 +7,75 @@ import NLink from 'next/link'
 import { colorMap } from './colorMap'
 import { MdClear } from 'react-icons/md'
 import { useRouter } from 'next/router'
+import { useIntl } from 'react-intl'
 
 
 const urlToDomain = (url) => new URL(url).hostname
 
-const CustomToolTip = React.memo(({ data, onClose }) => {
-  const theme = useTheme()
-  const { query: { probe_cc, axis_y } } = useRouter()
-  const dataKeysToShow = ['anomaly_count', 'confirmed_count', 'failure_count', 'ok_count']
+export const generateSearchQuery = (data, query) => {
+  const { since, until, axis_x, axis_y } = query
+
+  let sinceFilter = since
+  let untilFilter = until
+
+  if ('measurement_start_day' in data) {
+    sinceFilter = data.measurement_start_day
+    const untilPlus1 = new Date(Date.parse(sinceFilter))
+    untilPlus1.setUTCDate(untilPlus1.getUTCDate() + 1)
+    untilFilter = untilPlus1.toISOString().split('T')[0]
+  }
 
-  const linkToMeasurements = useMemo(() => {
+  const queryObj = ['probe_cc', 'test_name', 'category_code', 'probe_asn', 'input'].reduce((q, k) => {
+    if (k in data)
+      q[k] = data[k]
+    else if (k in query)
+      q[k] = query[k]
+    return q
+  }, {})
+  if (axis_y === 'input') {
+    queryObj.input = urlToDomain(data.input)
```

Comment:
You should not be converting the `input` into a `domain` for the search query. The search query works on `input` fields as well.

Reply:
This needs to be changed on the search page then. Since we only have UI input for `domain`, we filter out any extra URL params when constructing the API query.
e.g https://explorer.test.ooni.org/search?since=2022-01-25&until=2022-01-27&probe_cc=AF&test_name=web_connectivity&input=http%3A%2F%2Fbitcoin.org%2F&failure=false does not send the `input` param in the API call. Two changes can be made incrementally.
1. Allow `input` param to pass through to the API query
2. The above will show an empty `Domain` field but with results that are filtered. Some suitable UX solution to address this.

Reply:
Ah right. Well in that case let's just go for populating this with `domain`. The code above still needs to change as it should be using `domain` instead of input in the explorer URL.

Reply:
> should be using domain instead of input in the explorer URL